docs/pages/docs/query/sql-client.mdx

```diff
@@ -49,41 +49,27 @@ npm add @ponder/client
 
 Use `createClient` to connect to the server at the path where you registered the middleware.
 
-Import your schema from `ponder.schema.ts` into the same file. [Read more](#use-schema-from-a-different-repo) about schema portability.
-
 ```ts [Client project]
 import { createClient } from "@ponder/client";
-import * as schema from "../../ponder/ponder.schema";
-
-const client = createClient("http://localhost:42069/sql", { schema });
 
-export { client, schema };
+const client = createClient("http://localhost:42069/sql");
 ```
 
-### Run a query
+### Import `ponder.schema.ts`
 
-Use `client.db.select()` to build a custom SQL query using Drizzle and execute it over HTTP.
+Import your schema from `ponder.schema.ts`. [Read more](#use-schema-from-a-different-repo) about schema portability.
 
 ```ts [Client project]
-import { desc } from "@ponder/client";
-import { client, schema } from "../lib/ponder";
+import * as schema from "../../ponder/ponder.schema";
 
-const result = await client.db
-  .select()
-  .from(schema.account)
-  .orderBy(desc(schema.account.balance))
-  .limit(10);
+schema.account.address;
```

Comment:
I'm not sure I understand this code snippet. It looks somewhat incomplete without showing any usage of the `schema` object here in this Setup section.

Reply:
I replaced it with

```ts
import * as schema from "../../ponder/ponder.schema";

type Account = typeof schema.account.$inferSelect;
//   ^? { address: string; balance: bigint; }
```